co-authors-plus.php

```diff
@@ -654,15 +654,19 @@ function posts_join_filter( $join, $query ) {
 		return $join;
 	}
 
-	/**
-	 * Modify the author query posts SQL to include posts co-authored
-	 */
+    /**
+     * Modify the author query posts SQL to include posts co-authored
+     *
+     * @param string $where
+     * @param WP_Query $query
+     *
+     * @return string|string[]|null
+     */
 	function posts_where_filter( $where, $query ) {
 		global $wpdb;
 
 		if ( $query->is_author() ) {
 			$post_type = $query->query_vars['post_type'];
-			if ( 'any' === $post_type ) { 
```

Comment:
How come we are removing this line here?  This will cause a syntax error and throw off the support for `'any'` post types.

Reply:
I didn't think I was. 🤔
I think I may have messed up when selecting which bits to commit as I had a heap of white space changes show up due to my automatic tendency to run a code formatting shortcut after almost all edits and also doing this between other things.